info.json

```diff
@@ -0,0 +1,192 @@
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "tooltip": "The name of the certificate you would like to revoke."
+                }
+            ],
+            "output_schema": {
+                "kind": "",
+                "id": "",
+                "items": [],
+                "nextPageToken": "",
+                "selfLink": "",
+                "warning": {}
+            },
+            "open": false
+        }
+    ],
+    "active": true,
+    "cs_compatible": true,
+    "system": false,
+    "playbook_collections": [],
+    "icon_small": "data:image/jpeg;base64,iVBORw0KGgoAAAANSUhEUgAAAE0AAABJCAYAAAB4mKumAAAAIGNIUk0AAHomAACAhAAA+gAAAIDoAAB1MAAA6mAAADqYAAAXcJy6UTwAAAAJcEhZcwAADsQAAA7EAZUrDhsAAAABYktHRACIBR1IAAAAB3RJTUUH5wsNFBI4CfartwAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMy0xMS0xM1QyMDoxODo1NiswMDowMDEIb9oAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjMtMTEtMTNUMjA6MTg6NTYrMDA6MDBAVddmAAAAKHRFWHRkYXRlOnRpbWVzdGFtcAAyMDIzLTExLTEzVDIwOjE4OjU2KzAwOjAwF0D2uQAABYNJREFUeF7tm1+IVFUcx8/8c2Z2191VSzczaiUh0of+CIJB5QpCFJVRiyJCxEpg1hL1kGaBq64S9bAi9LD6YFFEKxhaEZuLoA++7JMsvkg9VJAJm7Os687uzp/md+fc5q4z997z53uud+V+YDjn3GHu3PO9v3N+v9+558bKFViEFHFeRkgQiaZAJJoCkWgKRKIpYMx7fnVplp26NMdbwbLp8QTb/2qGt/AYEW1z/xSv3V2GP2piiXiMt3DAh2dYBCO2HL3Na1igoo2MFXgtPBz6Ic9rOKCi9Z+d4bXwcOFqkddwwETbdcLMUEDw+gD22iCiFUtl9vuN8KawN6fK7HquxFv6QLxnmCZ/L0b2NfOaHtqWNvYnfs4wBcpRaVvaQrEyG4S1aVna8eHweUs/EA5LS7Qzo+GLy/wgh0WOSwdl0dBuPEh0MwUl0XIVF05ufCGj48CUHMFCm/zdUHUK0pYWxvxSFdW8VFo0dH459F7WuuOlYu1mUNvtGCAW/x/VvFRKNBP55dKW6iUUZyfniUQ0OlbI56DCqTg0YdGCyC/vhnDk0MixySDsCJCT/2fbM+zpzgRvqdN1+BaLxTArszT0RRGyNHR+iRCMQFqcjIMTsjR0iGHf1e4v/rZKWb7/4AGrfPaTv6wymWmHWJyotfmKRm4ZsfrZtbZmXR+/Un1SZHdalosHV1ll39C4VRIXr6VZmekJt3p5jA32NPGWO76iIaxsRVuMfftO/cXoiubk/JXbrP9cSdviRJ5geYpG7hiVLj23phZIfvrGMqtEWtr5K9NWiRiqfsPUVbTp2TJ76XNsXDY3fdMq7U47RWtrirNze1fy1nxOX55kx36e4K3Gv3eiK9xgT7YyVN19pOs3O7+s3jkkqewSXqsnu8i9k+3Nct5W16vuOuHdd1fR0KsYZPL0aTQfqULnsj9kqU5IuDuDYhRCcZou5AhM0/tiO6/VaJRNIAhEtH8mylZMZn9QOM/ZN/QvPzofE8K5irbv5TSvYRifaWXXc0Xrg8I+n985ZYV76hFvW3L9dvO6JK/h8HIEXh0f+CnHa+rICHd0m/c2Lc84jXLO3q/1N5BQrPjr3vrYBxnc7h68wcb+mOUtdxKLFrN4wt0gRPa2edrhuocwifX9reYdwWsbWnjNGz+LE9kM6JtG0ToaYp9XIT9RiZ2q+ylsS9G1NNXfE40srr87zTY86j8t+YpG0IotYgHSFq7R8FJBRzTiTuFEVzmEQg6RzF+EZKatkt4I/WUgOIfqjx+K91HI0ogzo3Ps+LD/RCtCc3ySZVLVv7XXxpzxm+gxVPiyZlUrO/l2K2/5I3zbt65P8Zo+U6XFVsDr7LQdb6kc00VGMEJqrHyzO8tr+oRlqL65qT798kPqqjvasZ0k4Q4OVZeLnJwcmWDjk/MtafS3fN0xBG91iYUqToTnNBtUCOKkMHOLlUvBv6jx3fsdbOVS+cxH2nRoKZiiZiTJdAuLxXFzpgg0NagIRkhbmo2JTTBBWtyFAw8qv82iPEmhV0GIoCzu+bVZrdd/lEUzsQpCBCFc37bqgx1VtNyhTBQtg0nhjuy4j9fU0RKNHoYsaTazgmFKuGce03+lUUs04nSvGWsj0ML9sr/xI0JZtEUjtq43M78RKOE6lydZUxrSXYxoe7bgPakThHCn3u3gNX0w0lcY2Kk/V3ihI9yeF+TzSy9goqGWxr0g4eJJ+UWD7o3y+aUXMNEI2nFjmkQqIyUc5ZdooKJRlG3SKdiICvfEwynl/NILqGgEOQW/h60I/IQjwY71rOAtLMoJux+0hHTk7IyRd8idFOfyrFSo7fJ5sjNdifqXwcKLRhgT7V7G/Di6B4lEUyASTYFINAUi0RSIRFMgEk2BSDQFItGkYew/HfFeETP8lUUAAAAASUVORK5CYII=",
```

Comment:
Remove value for icon_small  and icon_large  value of base64 encoded images content from